exporter/stefexporter/metadata.yaml

```
@@ -0,0 +1,13 @@
+type: stef
+
+status:
+  class: exporter
+  stability:
+    development: [metrics]
+  distributions: [contrib]
```

This should be empty until we publish it as part of https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
```suggestion
  distributions: []
```

---

Fixed.